### PR TITLE
README.md: Add netlify deploy instruction

### PR DESCRIPTION
**Pull request by RaiVaibhav**
Add instructions of deploying the netlify
on fork of a ``community`` repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,23 @@
...
+Deploy netlify on your forked ``community`` repository.
+After signup on [netlify](https://app.netlify.com/).
+
+- Click on **New site from Git**.
+- Choose **Git provider (Github)** > Choose a repository **< username >/community**.
+- Branch to deploy **master**.
+- Basic build settings :
+  * Set Build command to **.ci/build.sh**.
+  * Set Publish directory to **public**.
+  * Click Deploy site to deploy the site.
```
Comment (sks444) [inline README.md]: No need to click on the deploy button before setting the environment variables. There is a button name `Show advanced` after clicking on it you can add all of your environment variables, keys and values there.(e.g: in our case we need to put two keys: 1. GH_TOKEN and 2.OH_TOKEN) and then click on `Deploy site`.

File: README.md [hunk 2]
```diff
@@ -8,3 +8,23 @@
...
+- Basic build settings :
+  * Set Build command to **.ci/build.sh**.
+  * Set Publish directory to **public**.
+  * Click on **Show advanced >** **New variable** to add the `environment variables`.
+
+After that to set the ``environment variables``
+
+- Put **GH_TOKEN** in `Key` column and your generated `GitHub Access Token`
+  in `Value` column.
+- Similarly add others `variables` like **OH_TOKEN, GL_TOKEN** if needed.
```
Comment (sks444) [inline README.md]: others -> other

File: README.md [hunk 3]
```diff
@@ -8,3 +8,23 @@
 * It uses the [GCI private API](https://developers.google.com/open-source/gci/resources/downloads/TaskAPISpec.pdf)
 * __Attention:__ This repository only re-publishes publicly available information
 * Uses IGitt, supporting both GitHub & GitLab
+
+## Deploy Netlify on your fork :
+
+Deploy netlify on your forked ``community`` repository.
+After signup on [netlify](https://app.netlify.com/).
```
Comment (sks444) [inline README.md]: Include it in the steps.
```
- Sign up on [netlify](https://app.netlify.com/)
```

File: README.md [hunk 4]
```diff
@@ -8,3 +8,23 @@
...
+  * Set Build command to **.ci/build.sh**.
+  * Set Publish directory to **public**.
+  * Click on **Show advanced >** **New variable** to add the `environment variables`.
+
+After that to set the ``environment variables``
+
+- Put **GH_TOKEN** in `Key` column and your generated `GitHub Access Token`
+  in `Value` column.
+- Similarly add others `variables` like **OH_TOKEN, GL_TOKEN** if needed.
+- Click on **Deploy Site** to deploy the site.
```
Comment (sks444) [inline README.md]:
```
Click on **Deploy Site** to deploy the site.
```
Generally, we don't add full stops(`.`) while noting down points, you can remove it from the end of all the points. See our README.md

File: README.md [hunk 5]
```diff
@@ -8,3 +8,23 @@
...
+
+- Click on **New site from Git**.
+- Choose **Git provider (Github) >** Choose a repository **< username >/community**.
+- Branch to deploy **master**.
+- Basic build settings :
+  * Set Build command to **.ci/build.sh**.
+  * Set Publish directory to **public**.
+  * Click on **Show advanced >** **New variable** to add the `environment variables`.
+
+After that to set the ``environment variables``
```
Comment (sks444) [inline README.md]: use colon in the end.

File: README.md [hunk 6]
```diff
@@ -8,3 +8,23 @@
 * It uses the [GCI private API](https://developers.google.com/open-source/gci/resources/downloads/TaskAPISpec.pdf)
 * __Attention:__ This repository only re-publishes publicly available information
 * Uses IGitt, supporting both GitHub & GitLab
+
+## Deploy Netlify on your fork :
+
+Deploy netlify on your forked ``community`` repository.
```
Comment (sks444) [inline README.md]: use colon in the end?

Comment (sks444) [reply README.md]: Or maybe do something like:
```
Follow the following steps to deploy netlify on your fork of community repository:
```

File: README.md (hunk 4 above)
Comment (RaiVaibhav) [reply README.md]: Soory can't follow :stuck_out_tongue: , as wherever sentence ends, full stop will be there.

**Comment by jayvdb** [comment]
ack https://github.com/coala/community/pull/121/commits/632b9e103c953dd4c551a792e12b9abd0d2e978d

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**